examples/src/main/java/io/grpc/examples/helloworld/HelloWorldServer.java

```diff
@@ -89,7 +89,7 @@ public static void main(String[] args) throws IOException, InterruptedException
     server.blockUntilShutdown();
   }
 
-  private class GreeterImpl extends GreeterGrpc.GreeterImplBase {
+  static class GreeterImpl extends GreeterGrpc.GreeterImplBase {
```

Comment:
which you might expect? 
- rename the test class to `GreeterServiceImplTest` and `/** Unit tests for {@link HelloWorldServer.GreeterImpl} */ ` explicitly saying I'm testing service impl
- add a VisibleForTesting ServerBuilder setter or a constructor with `ServerBuilder` argument in `HelloWorldServer`, and modify `HelloWorldServer#start()` to consume a builder?